<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widgetLogo1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <widget class="QLabel" name="loginLogo1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../wfpics/logo1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetLogin" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>800</width>
     <height>701</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 235, 97);</string>
   </property>
   <widget class="QLabel" name="appName_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>561</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>56</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 51, 102);</string>
    </property>
    <property name="text">
     <string>WellFam Care</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="login">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>400</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#login{
	background-color: white;
	border: 2px solid rgb(0, 51, 102);
	border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEditUsername">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetUsername" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>300</width>
       <height>5</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 51, 102);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetPassword" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>300</width>
       <height>5</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 51, 102);</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowPassword">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="text">
      <string>Show Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signInButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton[active=&quot;true&quot;] {
	background-color: rgb(246, 235, 97);
	color:  rgb(0, 51, 102);
}

QPushButton {
	background-color: rgb(0, 51, 102);
	color: white;
	border-radius: 10px;
}

QPushButton:hover {
	background-color: #2980b9;
}
</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
